flake.nix

```diff
@@ -32,6 +32,26 @@
           pre-commit.settings.hooks.rustfmt.enable = true;
           pre-commit.settings.settings.rust.cargoManifestPath = "./rust/Cargo.toml";
 
+          # Check that we're using ///-style doc comments in Rust code.
+          #
+          # Unfortunately, rustfmt won't do this for us yet - at least not
+          # without nightly, and it might do too much.
+          pre-commit.settings.hooks.rust-doc-comments = {
+            enable = true;
+            files = ".*.rs$";
+            entry = "${pkgs.writeScript "rust-doc-comments" ''
```

Comment:
maybe `writeShellApplication`

Reply:
I'd do that for a larger or more critical script, but let's keep it lean for this one for now.